.travis.yml

```diff
@@ -0,0 +1,27 @@
+language: python
+
+python:
+  # We can add more version strings here when we support other python
+  # versions.
+  - "2.7"
+
+# Install or mock dependencies for alot as the package has to be imported
+# during generation of the docs.
+install:
+  - pip install configobj urwid twisted python-magic
+  # urwid has to be installed when installing urwidtrees otherwise the process
+  # fails.
+  - pip install urwidtrees
```

Comment:
Yes: our script imports alot (and hence all its dependencies). But we are able to mock them because they are not actually used. see https://github.com/pazz/alot/docs/source/generate_commands.py#L5